notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>I think we should start putting the <code>name</code> kwarg as the last one everyone.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>